deploy/helm-chart/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,312 @@
+{{- /* This file is generated using a script located at scripts/generate-helm-alerts.sh */}}
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "promscale.fullname" . }}-rules
+  namespace: {{ template "promscale.namespace" . }}
+  labels:
+    app: {{ template "promscale.fullname" . }}
+    chart: {{ template "promscale.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: "promscale-connector"
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/part-of: "promscale-connector"
+    app.kubernetes.io/component: "connector"
+spec:
+{{`
+  # Note: Alert thresholds are experimental. Feel free to change them or suggest back at
```

Comment:
Do we have to update this anytime we edit our alerts?

Reply:
yes